odl/discr/grid.py

```diff
@@ -123,7 +107,7 @@ def __init__(self, *coord_vectors, **kwargs):
         >>> g
         TensorGrid([1.0, 2.0, 5.0], [-2.0, 1.5, 2.0])
         >>> print(g)
-        grid [1.0, 2.0, 5.0] x [-2.0, 1.5, 2.0]
```

Comment:
Dunno, I seem to have had that idea once and got surprised by it.. I'll add it back.